components/scheduler/fx.go

```diff
@@ -23,6 +23,10 @@
 package scheduler
 
 import (
+	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/server/common/log"
+	"go.temporal.io/server/common/metrics"
+	"go.temporal.io/server/common/resource"
 	"go.uber.org/fx"
```

Comment:
nitpick: import groups should be third-party (including temporal api+sdk) then server
```suggestion
	"go.temporal.io/api/workflowservice/v1"
	"go.uber.org/fx"

	"go.temporal.io/server/common/log"
	"go.temporal.io/server/common/metrics"
	"go.temporal.io/server/common/resource"
```